showcase/src/app/home/_home.component.scss

```diff
@@ -0,0 +1,86 @@
+h1 {
+  font-size: 30px;
+}
+
+h2,
+.list {
+  font-size: 14px;
+  color: mat-color($grey-palette, 800);
+}
+
+h3 {
+  font-size: 19px;
+  color: mat-color($grey-palette, 800);
+}
+
+//custom title for the header
+.custom-about-title {
+  margin-left: 10px;
+  color: mat-color($grey-palette, 100);
+}
+
+//custom title for the section below the header
+.custom-title {
+  margin-bottom: 30px;
+  color: mat-color($grey-palette, 900);
+}
+
+//set the header image
+.about-background {
+  background: url("/assets/images/wallpaper/angular-banner.png") no-repeat center center;
+  background-size: cover;
+  height: 140px;
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+//defines a bottom line to separate all sections
+.section {
```

Comment:
We already have a `.stark-section` class, we should use this one. Then we can modify `.stark-section` style if needed and it will reflect to the entire app